src/applications/mhv-medical-records/containers/Allergies.jsx

```diff
@@ -230,7 +248,9 @@ ${allergies.map(entry => generateAllergyListItemTxt(entry)).join('')}`;
             <div className="vads-u-margin-y--5 vads-u-border-top--1px vads-u-border-color--white" />
           </>
         ) : (
-          <NoRecordsMessage type={recordType.ALLERGIES} />
+          !isLoadingAcceleratedData && (
```

Comment:
This is nit-picky but is there a way to check ` !isLoadingAcceleratedData` only once in this logic?